packages/components/src/Chip/Chip.css

```diff
@@ -1,9 +1,6 @@
-:root {
```

Comment:
Doing some cleanup while we're in here. We are moving any component-level variables that aren't meant to be shared off of `:root` to scope them to the component and clean up `:root`

Reply:
@ZakaryH Unless you can recall there being a reason for this

Reply:
It looks like it's safe to move these vars from `:root` to `.chip`, since I only found them being used it `Chip.css`. Generally, the CSS vars are declared in `:root` in case they need to be used globally.